tests/integration/golang/aim/dashboard/update_dashboard_test.go

```diff
@@ -75,8 +75,14 @@ func (s *UpdateDashboardTestSuite) Test_Ok() {
 				&resp,
 			)
 			assert.Nil(s.T(), err)
-			assert.Equal(s.T(), "new-dashboard-name", resp.Name)
-			assert.Equal(s.T(), "new-dashboard-description", resp.Description)
+
+			dashboards, err := s.dashboardFixtures.GetDashboards(context.Background())
```

Comment:
Possibly we should find the right Dashboard by ID (?) instead of trusting dashboards[0] to be the right one?

Reply:
That would mean adding a fixture to get dashboard by ID (alternative: a GET request). 
I can assert the length of the dashboards as 1 before the other assertions since I am creating only 1 dashboard

Reply:
well you are checking the ID match below at line 85, i think that's good.